src/webapp/js/reducers/locations_reducer.js

```diff
@@ -0,0 +1,13 @@
+import { FETCH_LOCATIONS } from '../actions/types';
+
+export default function (state = {}, action) {
```

Comment:
Will we use locations in some other places in the app? Are there in plans some other operations on locations or some api calls with locations? I'm just curious, if the name for reducer is good and reducer is necessary.

Reply:
I think we will need locations on module assignment screen